sql/atxs/atxs.go

```diff
@@ -811,3 +811,17 @@ func PoetProofRef(ctx context.Context, db sql.Executor, id types.ATXID) (types.P
 
 	return types.PoetProofRef(atx.NIPost.PostMetadata.Challenge), nil
 }
+
+func PositioningATX(ctx context.Context, db sql.Executor, id types.ATXID) (types.ATXID, error) {
+	var blob sql.Blob
+	if err := LoadBlob(ctx, db, id.Bytes(), &blob); err != nil {
+		return types.EmptyATXID, fmt.Errorf("get blob %s: %w", id, err)
+	}
+	// TODO: decide how to decode based on the `version` column
+	var atx wire.ActivationTxV1
+	if err := codec.Decode(blob.Bytes, &atx); err != nil {
+		return types.EmptyATXID, fmt.Errorf("decode %s: %w", id, err)
+	}
```

Comment:
Maybe the `sql/atxs` should just be in `activation/sql` instead?

Eventually, we will not get around it and we will need to decode ATXs to obtain other fields and having a single method reduces the chances of bugs.

Reply:
I think we will still need the separation of `activation` and `sql/atxs` since the `sql` packages should be able to be used by (most) other packages, while `activation` should only be a dependency of very few (`grpcserver`, `checkpoint` and `node` probably).